src/lib/core/components/visualizations/implementations/BoxplotVisualization.tsx

```diff
@@ -312,6 +315,73 @@ function BoxplotViz(props: VisualizationProps) {
       ? data.value?.completeCasesAllVars
       : data.value?.completeCasesAxesVars;
 
+  const dependentAxisRange: NumberOrDateRange | undefined = useMemo(() => {
+    if (isFaceted(data?.value)) {
+      // may not need to check yAxisVariable?.type but just in case
+      return data?.value?.facets != null &&
+        (yAxisVariable?.type === 'number' || yAxisVariable?.type === 'integer')
+        ? {
+            min:
+              (min([
+                0,
+                min(
+                  data.value.facets.flatMap((facet) =>
+                    facet.data.series
+                      .flatMap((o) => o.outliers as number[][])
+                      .flatMap((o) => o)
```

Comment:
You can use `.flat()` instead of `.flatMap((o) => o)`.

(Why is the method called `flat` and not `flatten`? [That's a long story...](https://developers.google.com/web/updates/2018/03/smooshgate))

Reply:
@jtlong3rd Ah, you are absolutely right! 👍 I thought I used flat() but did not change it. Updated 😃